content/en/docs/concepts/workloads/_index.md

```diff
@@ -8,31 +8,53 @@ no_list: true
 
 {{< glossary_definition term_id="workload" length="short" >}}
 Whether your workload is a single component or several that work together, on Kubernetes you run
-it inside a set of [Pods](/docs/concepts/workloads/pods).
-In Kubernetes, a Pod represents a set of running {{< glossary_tooltip text="containers" term_id="container" >}}
+it inside a set of [pods](/docs/concepts/workloads/pods).
+In Kubernetes, a `Pod` represents a set of running {{< glossary_tooltip text="containers" term_id="container" >}}
 on your cluster.
 
-A Pod has a defined lifecycle. For example, once a Pod is running in your cluster then
-a critical failure on the {{< glossary_tooltip text="node" term_id="node" >}} where that
-Pod is running means that all the Pods on that node fail. Kubernetes treats that level
-of failure as final: you would need to create a new Pod even if the node later recovers.
+`Pods` have a [defined lifecycle](/docs/concepts/workloads/pods/pod-lifecycle/).
+For example, once a `Pod` is running in your cluster then a critical fault on the
```

Comment:
Is it possible to use lowercase for some of the pod terms in lines 15-19?
Line 12 is clear.